repo/packages/H/hdfs/0/marathon.json

```diff
@@ -4,12 +4,15 @@
   "mem": 512,
   "disk": 100,
   "instances": 1,
-  "cmd": "{{#hdfs.custom-config}}echo {{hdfs.custom-config}} | base64 --decode > mesos-site.xml && {{/hdfs.custom-config}}cd hdfs-mesos* && if [ -e '../mesos-site.xml' ]; then cp -f ../mesos-site.xml ./etc/hadoop/; fi && ./bin/hdfs-mesos",
+  "env": {
+    "LD_LIBRARY_PATH": "{{hdfs.ld-library-path}}" 
+  },
```

Comment:
Print env is useful for debugging, but in the version we check in, we assume debugging is not required. I think we should be consistent and not display env here since we do that do this as a standard elsewhere. Also, it will keep the command a little simpler.